### DISCUSSION TITLE
Huggingface token in Official docker image

### DISCUSSION THREAD

**Discussion by Ofplarsen** [outlines-dev/outlines#1017] [Q&A]
Hey. How do I add my Huggingface token when running the official docker image? I can't download the correct model due to not being authenticated.

docker run -p 8000:8000 outlinesdev/outlines --model="mistralai/Mistral-7B-Instruct-v0.2"

**Comment by lapp0** [comment]
does setting `HF_TOKEN` in docker `--env` fix the problem?

**Reply by Ofplarsen** [reply]
I get the error that --env is not a known command

**Reply by Ofplarsen** [reply]
But it is fine, I found a work around (using my own docker)
